### ISSUE TITLE
"Space" seems to be allowed as accessible name for "Link has non-empty accessible name"

### ISSUE THREAD

**Issue by annethyme** [qualweb/act-rules#38]
On https://autismeplatform.dk/ I am seeing passes for links that according to Siteimprove an Accessibility inspector and Dev Tools are missing accessible name. 

My guess is that it is because the links have a space as their content. 
Stripped example: 
`<a href="#"> <img src="example.png" alt=""> </a>`

According to the followwing note in the Accessible name definition in the ACT-rule, I think it should be a fail: 
https://www.w3.org/WAI/standards-guidelines/act/rules/c487ae/#accessible-name 
"Note: As per the [accessible name and description computation](https://www.w3.org/TR/accname), accessible names are [flat string](https://www.w3.org/TR/accname-1.1/#terminology) trimmed of leading and trailing whitespace. Notably, it is not possible for a non-empty accessible name to be composed only of whitespace since these must be trimmed."

**Comment by BogdanCerovac** [comment]
I had a similar case and isolated the error.

It seems that `<noscript>` is the reason.

Minimum test case:

https://output.jsbin.com/rewilexafa/1

Test results: https://qualweb.di.fc.ul.pt/evaluator/https%253A%252F%252Fjsbin.com%252Frewilexafa%252F1

**Comment by carlosapaduarte** [comment]
Thanks for bringing this up @annethyme and for pointing it out @BogdanCerovac 

Fixed by https://github.com/qualweb/qualweb/pull/154
